Fixed examples used to get file paths.

antoinet opened the pull request:
The examples in the documentation/Readme used to refer to file paths with property `aff4path` in a GRR hunt do not work currently. Replaced them with property `pathspec.path`.

antoinet committed:
Fixed examples used to get file paths.

Karneades commented on README.md:
```diff
@@ -304,7 +304,7 @@ $ret.items.client_id | get-unique
 
 # Get unique file paths from a file finder hunt
-($ret.items.payload.stat_entry.aff4path).substring(31) | sort -u
+($ret.items.payload.stat_entry.pathspec.path).substring(31) | sort -u
```
substring(31) is not needed anymore because we filtered out the aff4 prefix using the substring function.

```suggestion
$ret.items.payload.stat_entry.pathspec.path | sort -u
```

Karneades reviewed (changes requested):
Thanks for the PR!

Karneades commented on docs/Get-GRRHuntResult.md:
````diff
@@ -24,7 +24,7 @@ Get hunt results for a specific hunt.
 ```
 PS C:\> $res = Get-GRRHuntResult -HuntId H:AAAAAAAA -Credential $creds
-PS C:\> ($res.items.payload.stat_entry.aff4path).substring(31) | sort -u
+PS C:\> ($res.items.payload.stat_entry.pathspec.path).substring(31) | sort -u
````
substring(31) is not needed anymore because we filtered out the aff4 prefix using the substring function.

```suggestion
PS C:\> $res.items.payload.stat_entry.pathspec.path | sort -u
```

Karneades commented on en-us/PowerGRR-help.xml:
```diff
@@ -3233,7 +3233,7 @@ False</dev:code>
         <maml:title>-------------------------- Example 1 --------------------------</maml:title>
         <dev:code>PS C:\&gt; $res = Get-GRRHuntResult -HuntId H:AAAAAAAA -Credential $creds
-PS C:\&gt; ($res.items.payload.stat_entry.aff4path).substring(31) | sort -u
+PS C:\&gt; ($res.items.payload.stat_entry.pathspec.path).substring(31) | sort -u
```
substring(31) is not needed anymore because we filtered out the aff4 prefix using the substring function.

```suggestion
PS C:\&gt; $res.items.payload.stat_entry.pathspec.path | sort -u
```

antoinet committed:
Apply suggestions from code review

substring(31) is not needed anymore because we filtered out the aff4 prefix using the substring function.

Co-Authored-By: Karneades <karneades@protonmail.com>
